### PR TITLE
haphazardly port everything to 1.19.4

### PR DESCRIPTION
**Pull request by sylv256**

### CODE REVIEW

**Comment by sylv256** [comment]
i read 1.19.4 as 1984 in my head

File: build.gradle [hunk 1]
```diff
@@ -30,8 +30,8 @@ dependencies {
 	modImplementation libs.quilt.loader
 	modImplementation libs.quilted.fabric.api
 
-	modImplementation "com.terraformersmc.terraform-api:terraform-wood-api-v1:4.2.0"
```
Comment (Oliver-makes-code) [inline build.gradle:33]: No more 420 :crab:

Comment (sylv256) [reply build.gradle:33]: sad

File: src/main/java/io/github/sweetberrycollective/wwizardry/mixin/Mixin_TerraformBoatTrackedData.java
Comment (Oliver-makes-code) [inline src/main/java/io/github/sweetberrycollective/wwizardry/mixin/Mixin_TerraformBoatTrackedData.java:1]: What's this for?

File: src/main/resources/wwizardry.accesswidener
Comment (Oliver-makes-code) [inline src/main/resources/wwizardry.accesswidener:1]: Why use an accesswidener when you have Mixin's `@Invoker`?

File: src/main/resources/wwizardry.mixins.json
Comment (Oliver-makes-code) [inline src/main/resources/wwizardry.mixins.json:1]: Why change spaces to tabs?

File: src/main/java/io/github/sweetberrycollective/wwizardry/item/WanderingItems.java [hunk 2]
```diff
@@ -1,53 +1,111 @@
...
+			WanderingDatagen.DENIA_WOOD.PRESSURE_PLATE_ITEM,
+			WanderingDatagen.DENIA_WOOD.SIGN_ITEM,
+			WanderingDatagen.DENIA_WOOD.SLAB_ITEM,
+			WanderingDatagen.DENIA_WOOD.STAIRS_ITEM,
+			WanderingDatagen.DENIA_WOOD.TRAPDOOR_ITEM,
+			WanderingDatagen.DENIA_WOOD.WOOD_ITEM,
+			WanderingDatagen.DENIA_WOOD.STRIPPED_LOG_ITEM,
+			WanderingDatagen.DENIA_WOOD.STRIPPED_WOOD_ITEM,
+	};
+	public static final List<ItemStack> STACKS = Arrays.stream(ITEMS).map(Item::getDefaultStack).toList();
```
Comment (Oliver-makes-code) [inline src/main/java/io/github/sweetberrycollective/wwizardry/item/WanderingItems.java]: Why? Why not automatically append items to this in the `registerItem` method?

File: src/main/java/io/github/sweetberrycollective/wwizardry/datagen/WoodType.java
Comment (Oliver-makes-code) [inline src/main/java/io/github/sweetberrycollective/wwizardry/datagen/WoodType.java:1]: The `blockSettings` copies were intentional, the methods self-modify

**Review by Oliver-makes-code — CHANGES_REQUESTED** [review]
Enjoy

File: src/main/java/io/github/sweetberrycollective/wwizardry/mixin/Mixin_TerraformBoatTrackedData.java
Comment (sylv256) [reply src/main/java/io/github/sweetberrycollective/wwizardry/mixin/Mixin_TerraformBoatTrackedData.java:1]: This fixes an error that occurs when the Terraform API registers Tracked Data via Vanilla which causes QSL to crash.

File: src/main/java/io/github/sweetberrycollective/wwizardry/datagen/WoodType.java
Comment (sylv256) [reply src/main/java/io/github/sweetberrycollective/wwizardry/datagen/WoodType.java:1]: oh oops

File: src/main/java/io/github/sweetberrycollective/wwizardry/mixin/Mixin_TerraformBoatTrackedData.java
Comment (Oliver-makes-code) [reply src/main/java/io/github/sweetberrycollective/wwizardry/mixin/Mixin_TerraformBoatTrackedData.java:1]: Ah

File: src/main/resources/wwizardry.mixins.json
Comment (sylv256) [reply src/main/resources/wwizardry.mixins.json:1]: that was the ide lol

Comment (sylv256) [reply src/main/resources/wwizardry.mixins.json:1]: it insists on using tabs :face_with_spiral_eyes:

**Pull request merged**